gapic/src/main/com/google/gapid/views/CommandTree.java

```diff
@@ -517,20 +558,24 @@ private void updateTable() {
     // with the minimum data requirement that is needed and referenced in GpuCountersDialog.
     return models.profile.getData().getGpuPerformance().getMetricsList().stream()
         .sorted((m1, m2) -> {
-          if (m1.getStaticAnalysis() != m2.getStaticAnalysis()) {
-            return m1.getStaticAnalysis() ? -1 : 1;
+          if (isStaticAnalysisCounter(m1) != isStaticAnalysisCounter(m2)) {
+            return isStaticAnalysisCounter(m1) ? -1 : 1;
```

Comment:
what is -1 and 1 here?

Reply:
This is a comparator. Take a look at https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Comparator.html#compare(T,T)